great_expectations/datasource/fluent/pandas_azure_blob_storage_datasource.pyi

```diff
@@ -142,11 +146,13 @@ class PandasAzureBlobStorageDatasource(_PandasFilePathDatasource):
     def add_excel_asset(
         self,
         name: str,
-        batching_regex: Union[re.Pattern, str],
-        container: str,
-        name_starts_with: str = "",
-        delimiter: str = "/",
+        *,
+        batching_regex: Union[re.Pattern, str] = ...,
+        batch_metadata: Optional[BatchMetadata] = ...,
```

Comment:
I usually put `batch_metadata` after `order_by`, but if it is too much work don't worry about it.

Reply:
I can do that

Reply:
Done